bluepysnap/morph.py

```diff
@@ -47,10 +48,7 @@ def __init__(self, morph_dir, population):
         # all nodes from a population must have the same model type
         if not self._is_biophysical(0):
             raise BluepySnapError("Node population does not contain biophysical nodes.")
-        self._load = nm.load_neuron
-        if MORPH_CACHE_SIZE is not None:
-            from functools import lru_cache
-            self._load = lru_cache(maxsize=MORPH_CACHE_SIZE)(self._load)
+        self._load = Morphology
```

Comment:
I'm looking at this strange function assigning, and don't see benefits. Should we just directly call `Morphology` in `get` function?

Reply:
I think you are right. I kept it because I was like : 
"ok this a bit better if we want to change the loader again latter on". 
But yes I don t find a lot of good usage for it. 

I will remove. Thanks!